lib/round.rb

```diff
@@ -0,0 +1,42 @@
+class Round
+  attr_reader :deck, :turns, :number_correct
```

Comment:
We don't need the `number_correct` here because you have defined this method below on line 27.